simpleclient_common/src/main/java/io/prometheus/client/exporter/common/TextFormat.java

```diff
@@ -100,8 +175,101 @@ private static String typeString(Collector.Type t) {
         return "summary";
       case HISTOGRAM:
         return "histogram";
+      case GAUGE_HISTOGRAM:
+        return "histogram";
```

Comment:
Hm, isn't this written out as a gauge? As per detecting if gsum/gcount is present?

Reply:
Histogram is the closest structurally, even if it's technically a gauge. The Python client does the same.